src/server/config.js

```diff
@@ -70,6 +71,13 @@ if (process.env.TIER_ACCESS_LEVEL) {
   config.tierAccessLevel = process.env.TIER_ACCESS_LEVEL;
 }
 
+// check whitelist is disabled
+if (config.encryptWhitelist === 'disabled') {
```

Comment:
I'm a bit on the fence about if it is okay to use `disabled` as a keyword to disable the whitelist, because what if user wants to use `disabled` as an actual keyword in the whitelist?

I guess it is okay because you can put it like `"encrypt_whitelist": [ "disabled" ]`, but you can also do sth like `"encrypt_whitelist": "bla"`.

Personally I would prefer to have another boolean `enable_encrypt_whitelist` field in config, so that we don't have any special words here. But I'm just being picky here 😈

Reply:
😂 cool ok.. haha